common/src/main/java/io/seata/common/thread/NamedThreadFactory.java

```diff
@@ -24,9 +26,10 @@
  * The type Named thread factory.
  *
  * @author jimin.jm @alibaba-inc.com
- * @date 2018 /9/12
+ * @author ggndnn
  */
 public class NamedThreadFactory implements ThreadFactory {
+    private final static Map<String, AtomicInteger> PREFIX_COUNTER = new ConcurrentHashMap<>();
```

Comment:
whether need specified capacity?

Reply:
I don' t think it's so necessary, the default value looks ok.